### ISSUE TITLE
Installation problem on windows 10/ubuntu-14.04.3-desktop-amd64 platform

### ISSUE THREAD

**Issue by php0id** [jonnnnyw/php-phantomjs#96]
After `composer require "jonnyw/php-phantomjs:4.*"` execution got error
```
> PhantomInstaller\Installer::installPhantomJS
Script PhantomInstaller\Installer::installPhantomJS handling the post-update-cmd event terminated with an exception


  [RuntimeException]
  Can not determine required version of jakoch/phantomjs-installer
```

If download zip-file end run `composer install` got error
```
> PhantomInstaller\Installer::installPhantomJS
Script PhantomInstaller\Installer::installPhantomJS handling the post-install-cmd event terminated with an exception


  [UnexpectedValueException]
  Invalid version string "~2.1"
```
Last error can be fixed by replacement in json-file:
`"jakoch/phantomjs-installer": "~2.1"`to `"jakoch/phantomjs-installer": "2.1.1"`

**Comment by psycholyzern** [comment]
You got the solution @php0id ?

**Comment by jonnnnyw** [comment]
I have just patched this in v4.3.2 which should solve this issue. Can you please confirm.

**Comment by psycholyzern** [comment]
Working great now. Thanks @jonnnnyw :)
